CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ subsections here on every PR where this is applicable.
 * Added support to skip staging artifact files during upload by selecting a storage policy by @ibindlish in https://github.com/wandb/wandb/pull/7142
 
 * `run.save()` accepts `pathlib.Path` values; by @timoffex in https://github.com/wandb/wandb/pull/7146
+* Added verbose and very-verbose mode to launch agent by @TimH98 in https://github.com/wandb/wandb/pull/7126
```

Comment:
This is the user facing doc, so maybe provide a bit more context what is the purpose of the verbosity or just say that you provide multiple verbosity levels